analyses/copy_number_consensus_call/src/scripts/merged_to_individual_files.py

```diff
@@ -0,0 +1,114 @@
+## Nhat Duong
+## November, 22 2019
+
+import numpy as np
```

Comment:
Small pep8 issue: import order should be standard first, then third party, then local.

Reply:
@cgreene for the 2500 CNVs question, we are considering these to be noisy/poor quality samples (this came from what GISTIC uses as a cutoff for noisy samples).